mgmt/rest/server.go

```diff
@@ -259,6 +304,18 @@ func (s *Server) addRoutes() {
 	s.addPprofRoutes()
 }
 
+func (s *Server) getAllowedOrigins(corsd string) []string {
+	if corsd == "" {
+		return []string{}
+	}
+	os := strings.Split(corsd, ",")
+	s.allowedOrigins = map[string]bool{}
+	for _, o := range os {
+		s.allowedOrigins[strings.TrimSpace(o)] = true
+	}
+	return os
```

Comment:
Consider adding more validation, for example if origin format is correct, something like: 
- httpa://domain.com
- 127.0.0.0.1
- http://?domain.com
- ...

should not pass, and user should be warned early that configuration is not valid.